Bulldozer.CSV/Maps/Financial.cs

```diff
@@ -1167,7 +1167,7 @@ private int ImportFinancialAccounts()
                 {
                     newAccount.Description = account.Name;
                 }
-                if ( account.Campus != null )
+                if ( account.Campus.CampusId.IsNotNullOrWhiteSpace() || account.Campus.CampusName.IsNotNullOrWhiteSpace() )
```

Comment:
@guymaness2 will this cause a null reference exception if account.Campus is not set? Should it be both? 
```suggestion
                if ( account.Campus != null && ( account.Campus.CampusId.IsNotNullOrWhiteSpace() || account.Campus.CampusName.IsNotNullOrWhiteSpace() ) )
```